### DISCUSSION TITLE
particle age

### DISCUSSION THREAD

**Discussion by spgerenday** [giorgk/ichnos#10] [General]
Hi,
Is AgeLimit in the configuration file in terms of time units or a number of time steps? Also, if an age limit is not specified, or if a streamline ends for a reason other than MAX_AGE, how can the age of the particle be determined?
Thanks

**Comment by giorgk** [comment]
The age limit is in time units. If there is no age limit the particle will continue until the particle exits the domain.
The output file contains the age of the particle in every location.
I just found that the documentation is not correct and I'll update this the next few hours and let you know
Thank you

**Comment by giorgk** [comment]
I have updated the documentation. Now it shows the correct output format
https://github.com/giorgk/ichnos/wiki/File-formats#output-file

**Comment by spgerenday** [comment]
Thank you
